src/pages/public_registration/DeletedPublicationInformation.tsx

```diff
@@ -0,0 +1,55 @@
+import { Registration, RegistrationStatus } from '../../types/registration.types';
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+interface DeletePublicationInformationProps {
+  registration: Registration;
+}
+
+export const DeletedPublicationInformation = ({ registration }: DeletePublicationInformationProps) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {(registration.status === RegistrationStatus.Deleted ||
+        registration.status === RegistrationStatus.Unpublished) && (
+        <Box
+          aria-hidden="true" // works in pair with DeletedPublicationScreenReaderInformation
+          sx={{
+            display: 'flex',
+            justifyContent: 'center',
+            backgroundColor: 'registration.main',
+            padding: '1rem',
+            gap: '1.5rem',
+            my: '1rem',
+          }}>
+          <Typography variant="h2" component="h1">
+            {t('registration.result_is_deleted')}
+          </Typography>
+        </Box>
+      )}
+    </>
+  );
+};
+
+export const DeletedPublicationScreenReaderInformation = ({ registration }: DeletePublicationInformationProps) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {(registration.status === RegistrationStatus.Deleted ||
+        registration.status === RegistrationStatus.Unpublished) && (
+        <Box
+          sx={{
+            position: 'absolute',
+            left: '-10000px',
+            width: '1px',
+            height: '1px',
+            overflow: 'hidden',
+          }}>
+          <Typography variant="h2" component="h1">
+            {t('registration.result_is_deleted')}
+          </Typography>
+        </Box>
+      )}
+    </>
+  );
```

Comment:
MUI hadde en `srOnly` for screen readers før, men ser nå at den er fjernet da: https://mui.com/material-ui/migration/v5-component-changes/#remove-sronly-variant  

```suggestion
  return registration.status === RegistrationStatus.Deleted ||
    registration.status === RegistrationStatus.Unpublished ? (
    <span style={visuallyHidden}>{t('registration.result_is_deleted')}</span>
  ) : null;
```

Reply:
litt usikker på hvordan det er med `<span>` vs `<h1>` etc her da. Mulig det helst skal være mer dominant enn `span` 🤔